products/harbor.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Harbor
+category: server-app
+tags: cncf docker registry
+iconSlug: harbor
+permalink: /harbor
+changelogTemplate: https://github.com/goharbor/harbor/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: End of general support
+
+auto:
+-   git: https://github.com/goharbor/harbor.git
+
+releases:
+-   releaseCycle: "2.9"
+    releaseDate: 2023-09-15
+    eol: 2024-09-30
+    latest: "2.9.1"
+    latestReleaseDate: 2023-09-30
+
+-   releaseCycle: "2.8"
+    releaseDate: 2023-05-22
+    eol: 2024-05-31
+    latest: "2.8.4"
+    latestReleaseDate: 2024-05-31
+
+-   releaseCycle: "2.7"
+    releaseDate: 2023-01-05
+    eol: 2024-01-31
+    latest: "2.7.4"
+    latestReleaseDate: 2023-11-30
```

Comment:
```suggestion
    latest: "2.7.3"
    latestReleaseDate: 2023-09-22
```

Reply:
Unfortunately, the VMWare webpage wasn't updated with latest release, 2.7.4 has been released recently : https://github.com/goharbor/harbor/releases/tag/v2.7.4

Reply:
Okay -> solved